reference-implementation/__tests__/resolving-not-yet-implemented.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+const { URL } = require('url');
+const { parseFromString } = require('../lib/parser.js');
+const { resolve } = require('../lib/resolver.js');
+const { BUILT_IN_MODULE_SCHEME } = require('../lib/utils.js');
+
+const mapBaseURL = new URL('https://example.com/app/index.html');
+const scriptURL = new URL('https://example.com/js/app.mjs');
+
+const BLANK = `${BUILT_IN_MODULE_SCHEME}:blank`;
+
+function makeResolveUnderTest(mapString) {
+  const map = parseFromString(mapString, mapBaseURL);
+  return specifier => resolve(specifier, map, scriptURL);
+}
+
+describe('Fallbacks that are not [built-in, fetch scheme]', () => {
+  const resolveUnderTest = makeResolveUnderTest(`{
+    "imports": {
+      "bad1": [
+        "${BLANK}",
+        "${BLANK}"
+      ],
+      "bad1-sub1": [
+        "${BLANK}/foo",
+        "${BLANK}"
+      ],
+      "bad1-sub2": [
+        "${BLANK}",
+        "${BLANK}/foo"
+      ],
+      "bad1-sub3": [
+        "${BLANK}/foo",
+        "${BLANK}/foo"
+      ],
+      "bad2": [
+        "${BLANK}",
+        "/bad2-1.mjs",
+        "/bad2-2.mjs"
+      ],
+      "bad2-sub": [
+        "${BLANK}/foo",
+        "/bad2-1.mjs",
+        "/bad2-2.mjs"
+      ],
+      "bad3": [
+        "/bad3-1.mjs",
+        "/bad3-2.mjs"
+      ]
+    }
+  }`);
+
+  it('should fail for [built-in, built-in]', () => {
+    expect(() => resolveUnderTest('bad1')).toThrow(/Not yet implemented/);
```

Comment:
"Not yet implemented" might be a little confusing, as it might mean "the support for this pattern of import maps is not yet implemented in the reference implementation" (which is what is intended here), or "the built-in module specified is not yet implemented". Adding a comment in this test might be helpful.